src/sentry/api/serializers/models/dsym_file.py

```diff
@@ -4,12 +4,11 @@
 
 from sentry.api.serializers import Serializer, register, serialize
 from sentry.models import (
-    ProjectDSymFile, GlobalDSymFile, VersionDSymFile, DSymApp
+    ProjectDSymFile, VersionDSymFile, DSymApp
```

Comment:
I want to remove it but I did not want to have a migration on this PR because it's already so big and in case we need to revert.